### PR TITLE
Scotland Class 5 - Tara Quinn - HTML/CSS Week One

### PR DESCRIPTION
**Pull request by Tara-Q**
-----
[View rendered zoo-css-challenge/README.md](https://github.com/Tara-Q/HTML-CSS-Challenges/blob/main/zoo-css-challenge/README.md)

### CODE REVIEW

File: zoo-css-challenge/index.html [hunk 1]
```diff
@@ -21,18 +21,30 @@
...
  		</section>
 
  		<div id="badge">
  			<p>Opening 2021!</p>
  		</div>
 
+		<section class="penguins">
+			<h2>Animal of the Month!</h2>
+			<h3>The Australian Little Penguin</h3>
+			<p>Only a tad bigger than a bowling pin, little penguins (sometimes called fairy penguins) are the smallest penguin species in the world.
```
Comment (bonboh) [inline zoo-css-challenge/index.html:37]: Whoa, these penguins are tiny! 🐧

File: zoo-css-challenge/style.css [hunk 2]
```diff
@@ -1,5 +1,6 @@
 * {
   font-family: sans-serif;
+  line-height: 2; /* is there a better way to do this? */
```
Comment (bonboh) [inline zoo-css-challenge/style.css:3]: This is the correct way to do this when we want to change the spacing between lines of text of the same paragraph 👍

File: zoo-css-challenge/style.css [hunk 3]
```diff
@@ -9,6 +10,13 @@ header {
   padding: 20px;
 }
 
+#logo {
+  border-width: 0;
+}
```
Comment (bonboh) [inline zoo-css-challenge/style.css:15]:
```suggestion
```
Alternatively, we can delete the `img` selector below on line 40 which adds the border

```diff
-img {
-  border: 5px solid #8075ff;
-}
```

File: zoo-css-challenge/style.css [hunk 4]
```diff
@@ -64,6 +72,14 @@ button {
...
   border: 2px solid black;
   color: #fff;
+  font-size: 125%;
+}
+button:hover {
+  background-color:rgb(136, 136, 185);
+}
+
+button:focus {
+  background-color:rgb(136, 136, 185);
```
Comment (bonboh) [inline zoo-css-challenge/style.css:82]:
```suggestion
button:hover,
button:focus {
  background-color:rgb(136, 136, 185);
```
As these two selectors have the same CSS declaration, we can combine them like so

File: zoo-css-challenge/style.css [hunk 5]
```diff
@@ -184,12 +235,19 @@ footer {
   color: #ff3f00;
 }
 
+.membership p {
+  max-width: 700px;
+  margin-left: 100px;
```
Comment (bonboh) [inline zoo-css-challenge/style.css:240]:
```suggestion
  margin: auto;
```
Alternatively, we can use `margin: auto` which centers the element by distributing the margins around equally

File: zoo-css-challenge/notes: [hunk 6]
```diff
@@ -0,0 +1,5 @@
+notes:
+changing borders on bear images - could i add ids to each of the bear images and style them that way or would a better way be 
+to use a css selector like 'first-of-type' within the class of image-container?
```
Comment (bonboh) [inline zoo-css-challenge/notes::3]: This exercise wants you to use `first-of-type`/`nth-child(2)` to practice using those kinds of selectors. 

As mentioned in class, software teams typically prefer using class selectors so they would be preferred over ID selectors 👍

File: zoo-css-challenge/notes: [hunk 7]
```diff
@@ -0,0 +1,5 @@
...
+
+membership margin: have set it to 100px left but possibly best to use rem??
```
Comment (bonboh) [inline zoo-css-challenge/notes::5]: Yes, for margins we should use `rem` over `px`. The answer the exercise is looking for is `margin: auto`, but I realise we actually didn't cover `auto` in class 😅

**Review by bonboh — APPROVED** [review]
Great job @Tara-Q! I've left a few suggestions and tips for you to read, but there's nothing to do otherwise 😄

File: zoo-css-challenge/style.css (hunk 3 above)
Comment (Tara-Q) [reply zoo-css-challenge/style.css:15]: Ahh yes! Thanks for the tips and for reviewing my work.

**Pull request closed without merging**